app/controllers/carts_controller.rb

```diff
@@ -16,7 +16,15 @@ def show
     @items = Item.find(@cart.contents.keys)
   end
 
-
+  def destroy
+    # require 'pry', binding.pry
+    # item = Item.find(session[:cart].select{|k,v|k = :id}.keys)[0]
+    item = Item.find(params[:format])
+    @cart.contents.delete(params[:format])
+    flash[:successfully_removed] = "Successfully removed <a href=#{item_path(item)}>#{item.title}</a> from your cart."
```

Comment:
How does this get rendered as a link, not text?

Reply:
in the application.html.erb file, we had to include a raw(msg) in our flash message so that it renders the raw html correctly. I tried a few different ways but the href ended up being the best solution over a link_to rails method